Sources/Kevin/Core/Styles/KevinTheme.swift

```diff
@@ -97,9 +97,27 @@ open class KevinTheme {
         public var shadowOffset = CGSize(width: 1.0, height: 1.0)
         public var shadowColor = UIColor(red: 1, green: 0, blue: 32/255, alpha: 1)
         
+        public var negativeBackgroundColor = UIColor(red: 230/255, green: 231/255, blue: 238/255, alpha: 1)
```

Comment:
maybe creating separate style NegativeButtonStyle inheriting part of the MainButtonStyle attributes would be better.

Reply:
Yes, it would be, but there is a problem with structs - they don't really inherit one another:
```
public struct NegativeButtonStyle: MainButtonStyle { } -> Inheritance from non-protocol type 'KevinTheme.MainButtonStyle'
```
Basically you only can use protocols on structs.

We can change struct to class, so we can use inheritance. So hence the question - why structs in first place? And what will be downsides of using classes instead?

Reply:
I did not mean inheritance as per say, just copy the properties.